src/services/twilio-help/twilio-command-help.js

```diff
@@ -31,6 +33,45 @@ class TwilioCommandHelp extends CommandHelp.default {
     return returnList.join('\n');
   }
 
+  // To add the API help document url
+  docs() {
+    const listOfDetails = [];
+    const helpDoc = this.command.docLink || getDocLink(this.command.id);
+    if (helpDoc !== undefined) {
+      listOfDetails.push(chalk.bold('MORE INFO'));
+      listOfDetails.push(indent(helpDoc, 2));
+    }
+    return listOfDetails.join('\n');
+  }
+
+  // overriding to include docs()
+  generate() {
+    const cmd = this.command;
+    const flags = util.sortBy(
+      Object.entries(cmd.flags || {})
+        .filter(([, v]) => !v.hidden)
+        .map(([k, v]) => {
+          v.name = k;
+          return v;
+        }),
+      (f) => [!f.char, f.char, f.name],
+    );
+    const args = (cmd.args || []).filter((a) => !a.hidden);
+    let output = util
+      .compact([
+        this.usage(flags),
+        this.args(args),
+        this.flags(flags),
+        this.description(),
+        this.aliases(cmd.aliases),
+        this.examples(cmd.examples || cmd.example),
+        this.docs(),
```

Comment:
Test for this change is missing in twilio-help.test. Can we add tests to make sure the docs are displayed when we pass --help command. Something similar to what @onuzbee did for OPTIONAL/REQUIRED flag

Reply:
Did this [here](https://github.com/twilio/twilio-cli/blob/44ccef891dffd9501e9950eeeb143dcd093e7fad/test/services/twilio-help/twilio-help-doc.test.js#L42) .